app/views/catalog/_document_media_object.atom.builder

```diff
@@ -19,9 +19,8 @@ xml.entry do
 
   with_format(:html) do
     xml.summary "type" => "html" do
-      xml.text! render_document_partial(document,
-      :index,
-      document_counter: document_counter)
+      document_component = blacklight_config.view_config(:atom).summary_component
+      render document_component.new(document: document_presenter(document), component: :div, show: true)
```

Comment:
Does this need `xml.text!` to write out the rendered content?

Reply:
Yep. I assumed the html in the output was a improper data being returned because they'd changed things and so removed `xml.text!` because that had better formatted output. But looking at the upstream builder and output from MCO it is supposed to have the html in the output.

Having put the `xml.text!` back into place though, I'm seeing that there is a difference in the html being rendered out. So need to dig into that.